way-cooler/src/shells/xdg.rs

```diff
@@ -1,7 +1,7 @@
 use std::rc::Rc;
 
 use wlroots::{CompositorHandle, Origin, SurfaceHandle, SurfaceHandler,
-              XdgShellSurfaceHandle, XdgShellHandler, XdgShellManagerHandler};
+              XdgShellSurfaceHandle, XdgShellHandler, XdgShellManagerHandler, XdgShellState::*};
```

Comment:
Don't do a glob import for enums at the top level, put this use statement in the function you use it (e.g `map_request`)